packages/devtools_app/lib/src/debugger/flutter/codeview.dart

```diff
@@ -422,29 +438,98 @@ class LineItem extends StatelessWidget {
   const LineItem({
     Key key,
     @required this.lineContents,
-    @required this.isPausedHere,
+    this.pausedFrame,
   }) : super(key: key);
 
   final String lineContents;
-  final bool isPausedHere;
+  final StackFrameAndSourcePosition pausedFrame;
 
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
 
-    final regularStyle = TextStyle(color: theme.textTheme.bodyText2.color);
-    final selectedStyle = TextStyle(color: theme.textSelectionColor);
+    Widget child;
+    if (pausedFrame != null) {
+      final column = pausedFrame.column;
 
-    return Container(
-      alignment: Alignment.centerLeft,
-      height: CodeView.rowHeight,
-      color: isPausedHere ? theme.selectedRowColor : null,
-      child: Text(
+      final foregroundColor =
+          isDarkTheme ? theme.textTheme.bodyText2.color : theme.primaryColor;
+
+      // The following constants are tweaked for using the
+      // 'Icons.label_important' icon.
+      const colIconSize = 13.0;
+      const colLeftOffset = -3.0;
+      const colBottomOffset = 13.0;
+      const colIconRotate = -90 * math.pi / 180;
+
+      child = Stack(
+        children: [
+          Text(
+            lineContents,
+            maxLines: 1,
+            overflow: TextOverflow.ellipsis,
+          ),
+          Row(
+            children: [
+              Text(' ' * (column - 1)),
+              Transform.translate(
```

Comment:
consider tweaking the background color of all text after the column rather than drawing the arrow.